README.md

```diff
@@ -1 +1,19 @@
 # freebsd-netlink
+
+
+Design document is here
+https://docs.google.com/document/d/1VGci3zucEhCphwLkCPVFjLkMudW2vdZUkgjTyqOSaCU/edit#
+
+
+## Note:
+
+### About headers:
+As per common practice, I have seperated the userspace API from the kernel API (that has access to the userspace API). I kept it simple, and I kept the userspace api in linux/netlink.h and the kernel api in net/netlink.h
```

Comment:
I'm changing the folder in the next PR. 

uapi/netlink.h for userspace information
and 
net/netlink.h for both userspace nad kernel